src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -303,11 +307,8 @@ protected boolean compileAdditionalSources()
 
         processRunner.addArg("-I" + workDir.toString());
 
-        for (String fileName : getAdditionalSourceFiles()) {
-            FileOps.copyResource(getAdditionalSourceFileLocation()  + fileName, workDir.resolve(fileName));
-            processRunner.addArg(fileName);
-        }
-        
+        processRunner.addArgs(copyAdditionalSourceFiles(workDir));
```

Comment:
This has nothing to do with passing runtimeArgs, does it?
I'm not sure the addArg(String)  is exactly the same as addArgs(List<String>) but if you tested it, I'm ok.

Reply:
Got it, I see now what it has to do with runtimeArgs